docs/modules/ROOT/pages/clc-viridian.adoc

```diff
@@ -62,3 +66,68 @@ Usage:
 ----
 clc viridian list-clusters [flags]
 ----
+
+== clc viridian list-custom-classes
+
+Lists all custom classes in the Viridian Cluster.
+
+Make sure you login before running this command.
+
+Usage:
+
+[source,bash]
+----
+clc viridian list-custom-classes <cluster-name/cluster-id> [flags]
```

Comment:
Let's copy this line from command's usage to be consistent.
Same comment for other usage lines in the doc.

Reply:
What do you mean, I didn't understand?

Reply:
ok, updated docs and help texts to be more consistent